examples/example.py

```diff
@@ -0,0 +1,23 @@
+import poe
+import logging
+
+poe.logger.setLevel(logging.INFO)
+client = poe.Client("SqX7oTPz0qV1GKgkLEsCmA%3D%3D")
+
+import time
+
+request = 300
+wait_time = 600 / request
```

Comment:
Also, it'd be better to add the wait functionality to `examples/repeated_messages.py` instead of creating a new file.